lib/krane/cli/base.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'thor'
+
+module Krane
+  class CLI < Thor
+    package_name "Krane"
+
+    def self.exit_on_failure?
+      true
+    end
+
+    private
+
+    def logger(verbose: false)
+      @logger ||= KubernetesDeploy::FormattedLogger.build(@namespace, @context, verbose_prefix: verbose)
```

Comment:
Not sure if it's better to keep the base `krane` command without `@namespace, @context` and override as needed in each command or just do it all here.

Reply:
🤞 this method goes away with a TaskContext object.